state.go

```diff
@@ -569,7 +575,7 @@ func (a *AuthorizedState) Handle(ctx *StateContext, c Controller) (State, bool)
 	// handle easy case first, no update info present, means no update in progress
 	if err != nil && os.IsNotExist(err) {
 		log.Debug("no update in progress, proceed")
-		return updateCheckWaitState, false
+		return inventoryUpdateState, false
```

Comment:
Agree for now. What I've been thinking (and it is actually my bad that I am late to the party) is that in future maybe we will need some configuration option, how frequently you will need to update inventory (same as how often do you want to ask for new updates).
Then we will have to refactor this as there is no chance we will be able to do so (interval handling for inventory sending).